<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rotinas</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="server_label">
      <property name="text">
       <string>Servidor :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="server_input"/>
     <widget class="QPushButton" name="load_btn">
      <property name="text">
       <string>&amp;Carregar</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="routines_area">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>527</width>
        <height>325</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Pesquisar :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_input"/>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QProgressBar" name="routines_progress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="run_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Executar</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
